python/paddle/fluid/distribute_transpiler.py

```diff
@@ -500,38 +484,83 @@ def _clone_var(self, block, var):
             lod_level=var.lod_level,
             persistable=True)
 
-    def _append_split_op(self, program, gradblocks):
+    def _find_op_by_out(self, program, var):
+        for idx, op in enumerate(program.global_block().ops):
+            if var.name in op.output_arg_names:
+                return idx + 1
```

Comment:
Maybe `return idx` here and the caller do `insert_idx = self._find_op_by_out(program, splited_vars[0]) + 1`, since the name of this function indicates the index is the index of the op, not the index of the op to be inserted.

Reply:
Done.